### PR TITLE
add hint when no farms

### PR DESCRIPTION
**Pull request by amiraabouhadid**
### Description

added a hint to the user to change key types from TFChain wallet when he/she cannot see their farms. 

### Related Issues

#1707

image

### CODE REVIEW

File: packages/playground/src/dashboard/components/user_farms.vue [hunk 1]
```diff
@@ -7,6 +7,9 @@
       single-line
       hide-details
     ></v-text-field>
+    <v-alert type="info" variant="tonal" v-if="farmsCount == 0">
```
Comment (MohamedElmdary) [inline packages/playground/src/dashboard/components/user_farms.vue:10]: what do u think about styles should we add margin bottom?
@amiraabouhadid @ehab-hassan

Comment (ehab-hassan) [reply packages/playground/src/dashboard/components/user_farms.vue:10]: yes let's add some margin-bottom and I think a warning color here like this
image

Comment (amiraabouhadid) [reply packages/playground/src/dashboard/components/user_farms.vue]: done, thanks

**Comment by AlaaElattar** [comment]
- I don't understand the point of the hint. maybe the user has no farms, why to mention keytype ?

**Review by ehab-hassan — APPROVED** [review]
Approved at design

**Comment by amiraabouhadid** [comment]
> * I don't understand the point of the hint. maybe the user has no farms, why to mention keytype ?

please refer to #1113

**Pull request merged**